src/RageUtil.cpp

```diff
@@ -1195,6 +1195,9 @@ float calc_mean(const float* pStart, const float* pEnd)
 	/* The Kahan summation algorithm is used here to prevent
 	 * situations where the low order bits may be lost.
 	 * https://en.wikipedia.org/wiki/Kahan_summation_algorithm */
+
+	ASSERT_M( pStart != pEnd, "calc_mean called with empty range" );

```

Comment:
I think if they're equal then can just return 0.0 instead of asserting.

Reply:
sounds good